circe/pricing_push/compute.py

```diff
@@ -774,46 +793,71 @@ def process_IN_CLOSE_WRITE(self, event):
         flag = next_mul[key][0]
 
         if next_tasks_map[task_name][0] in home_ids: 
-            print('----- next step is home')
+            logging.debug('----- next step is home')
             transfer_data(home_id,username,password,event.pathname, "/output/"+new_file)   
         else:
-            print('----- next step is not home')
+            logging.debug('----- next step is not home')
             for next_task in next_tasks_map[task_name]:
                 next_key = (next_task,mapping_input_id[(home_id,input_name)])
                 while next_key not in task_node_map:
-                    print('Not yet loaded assignment')
+                    logging.debug('Not yet loaded assignment')
                     time.sleep(1)
 
-            print('Loaded all required assignment')
+            logging.debug('Loaded all required assignment')
             next_hosts =  [task_node_map[x,mapping_input_id[(home_id,input_name)]] for x in next_tasks_map[task_name]]
 
-            print('Sending the output files to the corresponding destinations')
+            logging.debug('Sending the output files to the corresponding destinations')
             if flag=='true': 
-                print('send a single output of the task to all its children') 
+                logging.debug('send a single output of the task to all its children') 
+
+                #Unicast
```

Comment:
Remove commented code.

Reply:
Same reason above for unicast/ multicast.